app/src/main/kotlin/org/fossify/musicplayer/activities/MainActivity.kt

```diff
@@ -141,7 +140,8 @@ class MainActivity : SimpleMusicActivity() {
         }
 
         binding.mainMenu.onSearchTextChangedListener = { text ->
-            getCurrentFragment()?.onSearchQueryChanged(text)
+            val normalizedText = text.normalizeString()
+            getCurrentFragment()?.onSearchQueryChanged(normalizedText)
```

Comment:
Normalize query directly in fragments so we can pass the raw text to the adapters for highlighting.

```suggestion
            getCurrentFragment()?.onSearchQueryChanged(text)
```